prov/efa/src/dgram/efa_ep.c

```diff
@@ -374,50 +206,31 @@ static int efa_ep_setflags(struct fid_ep *ep_fid, uint64_t flags)
 	return 0;
 }
 
-/* efa_ep_create_self_ah() create an address handler for
- * an EP's own address. The address handler is used by
- * an EP to read from itself. It is used to
- * copy data from host memory to GPU memory.
- */
-static inline
-int efa_ep_create_self_ah(struct efa_ep *ep, struct ibv_pd *ibv_pd)
-{
-	struct ibv_ah_attr ah_attr;
-	struct efa_ep_addr *self_addr;
-
-	self_addr = (struct efa_ep_addr *)ep->base_ep.src_addr;
-
-	memset(&ah_attr, 0, sizeof(ah_attr));
-	ah_attr.port_num = 1;
-	ah_attr.is_global = 1;
-	memcpy(ah_attr.grh.dgid.raw, self_addr->raw, sizeof(self_addr->raw));
-	ep->base_ep.self_ah = ibv_create_ah(ibv_pd, &ah_attr);
-	return ep->base_ep.self_ah ? 0 : -FI_EINVAL;
-}
-
 static int efa_ep_enable(struct fid_ep *ep_fid)
 {
 	struct ibv_qp_init_attr_ex attr_ex = { 0 };
 	struct ibv_pd *ibv_pd;
 	struct efa_ep *ep;
-	int err;
 	ep = container_of(ep_fid, struct efa_ep, base_ep.util_ep.ep_fid);
 
 	if (!ep->scq && !ep->rcq) {
-		EFA_WARN(FI_LOG_EP_CTRL,
-			 "Endpoint is not bound to a send or receive completion queue\n");
+		EFA_WARN(
+			FI_LOG_EP_CTRL,
```

Comment:
Why add a new line here?

Reply:
I used VS Code to format according to the Linux kernel's `.clang-format` file and it added a line. Removed.